### DISCUSSION TITLE
Check environment in typescript file

### DISCUSSION THREAD

**Discussion by elenaaralla** [serenity-is/Serenity#6830] [General]
Hi all,

I've added a script in  _Modules\Membership\Account\Login\LoginPage.ts_ that fill in username and password via jquery and submit the login form, to skip the login in developement.

How can I check in TS files what is the environment, so that this script runs only when the application in in DEBUG mode?

Something similar to c#

#IF DEBUG
#ENDIF

Thank you,
Elena.

**Comment by kilroyFR** [comment]
it's not a serenity question but a typescript question. 
There are some TS plugins (nugligy or https://lisperator.net/uglifyjs/compress).
In former 3.x version of serenity nuglify was used. I did not check on 6.x versions
